framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -366,6 +381,121 @@ public S initialStore(UnderlyingAST underlyingAST, List<LocalVariableNode> param
     return store;
   }
 
+  /**
+   * Determine whether a lambda expression is leaked.
```

Comment:
Currently this is quite limited, handling only lambdas that are immediately passed to a method.  This is a reasonable limitation, but please document it.